src/shared/src/backend-supports/project-supports.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { WbsElementStatus } from '../types/project-types';
+import { WBS_Element_Status } from '@prisma/client';
```

Comment:
im pretty sure shared cant import from the prisma client because of how we deploy stuff so all of this should be moved to project utils file in the backend